nemo/collections/llm/README.md

```diff
@@ -0,0 +1,13 @@
+NeMo LLM Collection
+===================
+
+The NeMo LLM Collection is the first collection that features NeMo 2.0, a redesign of NeMo that
+aims to improve the user experience in by taking a more PyTorch Lightning-like approach to make
+NeMo both simpler and more modular. 
+
+The current models are currently reimplemented in 2.0 as part of this collection:
+- **GPT**
+- **LLaMA**
+- **Mixtral**
+
+For detailed tutorials and documentation of NeMo 2, please refer to the NeMo 2.0 [docs](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo_2.0/index.html) (TODO: update this hyperlink).
```

Comment:
Do not forget to remove TODO internal note. 

For detailed tutorials and documentation on NeMo 2.0, refer [docs](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo_2.0/index.html).